src/Apps/Order2/Routes/Checkout/Components/FulfillmentDetailsStep/Order2PickupForm.tsx

```diff
@@ -11,12 +11,15 @@ import { validateAndExtractOrderResponse } from "Apps/Order/Components/ExpressCh
 import { useCheckoutContext } from "Apps/Order2/Routes/Checkout/Hooks/useCheckoutContext"
 import { useOrder2SetOrderFulfillmentOptionMutation } from "Apps/Order2/Routes/Checkout/Mutations/useOrder2SetOrderFulfillmentOptionMutation"
 import { useOrder2SetOrderPickupDetailsMutation } from "Apps/Order2/Routes/Checkout/Mutations/useOrder2SetOrderPickupDetailsMutation"
-import { richPhoneValidators } from "Components/Address/utils"
+import {
+  richPhoneValidators,
```

Comment:
Remember, I created in a previous PR the richRequiredPhoneValidators. I think the phone number in the pick-up is mandatory. 

(you might have to rebase main)